packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -386,13 +411,16 @@ export class TokenRatesController extends BaseController<
   }
 
   /**
-   * Gets current chain ID slug from cached supported platforms CoinGecko API response.
+   * Gets the slug from cached supported platforms CoinGecko API response for the chain ID.
    * If cached supported platforms response is stale, fetches and updates it.
    *
-   * @returns The CoinGecko slug for the current chain ID.
+   * @param chainId - The chain ID.
+   * @returns The CoinGecko slug for the chain ID.
    */
-  async getChainSlug(): Promise<string | null> {
-    const { threshold, chainId } = this.config;
+  async getChainSlug(
```

Comment:
Another one we can make private.

Reply:
I also don't see this one referenced in the test file

Reply:
https://github.com/MetaMask/core/blob/1e77d0144008a44e537808ec5547a4dbb512dab4/packages/assets-controllers/src/TokenRatesController.test.ts#L1186

Reply:
Oh you're adding them in your PR... I was looking on `main`... so yea I'm sorry cuz you put in the effort to create tests for these but really they are essentially private methods that just weren't marked as such and I feel like we should mark them as private now. Lets see what @mcmire and @Gudahtt think?

Reply:
Less API surface area generally sounds good to me!

Alternatively, if we did want proper test coverage of something we could pull it out into a separate module, and test it that way. Then it can still be private, but will follow our convention of reducing API surface area and not testing private things.

The guideline of not testing private methods is really a nudge to organize code into more cohesive chunks.

Reply:
Lets make this private in a subsequent PR per @mcmire 's[ suggestion](https://consensys.slack.com/archives/C063W9EE484/p1700166880644369)